.github/workflows/benchmarks.yml

```diff
@@ -12,7 +12,7 @@ env:
 
 jobs:
   benchmarks:
-    runs-on: macos-14
+    runs-on: macos-15
```

Comment:
I didn't check all the workflows, I'll keep an eye on them in the coming days, please let me know if you see anything strange